core/neat/functions/_retrieve-neat-settings.scss

```diff
@@ -0,0 +1,16 @@
+@charset "UTF-8";
+/// Return a Neat setting.
+///
+/// @argument {string} $setting
+///
+/// @return {boolean | color | list | number | string}
+///
+/// @example scss
+///   _retrieve-neat-setting(columns)
+///
+/// @access private
+
+@function _retrieve-neat-setting($_grid, $setting) {
```

Comment:
Can we update the name of the partial to match the name of the function? The file name currently has an `s` at the end.